### ISSUE TITLE
.Net Bug: OpenAPI descriptions with parameters defined under the path item are ignored

### ISSUE THREAD

**Issue by calebkiage** [microsoft/semantic-kernel#8268]
**Describe the bug**
According to the [OpenAPI spec](https://spec.openapis.org/oas/v3.0.3#fixed-fields-6), parameters are allowed under the path object and should apply to all operations under that path item. Semantic Kernel currently only uses parameters under the operation object.

An example OpenAPI:

```json
{
  "openapi": "3.0.3",
  "info": {
    "title": "Test Schema",
    "version": "0"
  },
  "servers": [
    {"url": "https://localhost:3000"}
  ],
  "paths": {
    "/users/{userId}": {
      "get": {
        "operationId": "getUserById",
        "responses": {
          "default": {
            "description": ""
          }
        }
      },
      "parameters": [
        {
          "name": "userId",
          "in": "path",
          "required": true
        }
      ]
    }
  }
}
```

**Expected behavior**
Semantic kernel should consider path item parameters as well as operation parameters when building a plugin

**Platform**
 - OS: Windows
 - IDE: VS Code
 - Language: C#
 - Source: main branch of repository

**Additional context**

**Comment by markwallace-microsoft** [comment]
Hi @calebkiage thanks for creating this issue. Is this a blocking issue for you and if yes, can you  provide some detail on the impact to help us to prioritise this.

**Comment by calebkiage** [comment]
Hi @markwallace-microsoft, it's not a blocking issue. I'm cataloguing gaps in semantic kernel's OpenAPI support and opening issues for tracking.

**Comment by markwallace-microsoft** [comment]
The SK Open API support does not cover 100% of the Open API specification. As features are requested we aim to provide a functionality very quickly, typically this happens in time for our next weekly release. Closing this issue until we get a customer request where this is a blocking issues.

**Comment by baywet** [comment]
For other readers, this was fixed by #9969
